dotnet/samples/Hello/Hello.AppHost/Program.cs

```diff
@@ -5,15 +5,24 @@
 
 var builder = DistributedApplication.CreateBuilder(args);
 var backend = builder.AddProject<Projects.Backend>("backend").WithExternalHttpEndpoints();
-builder.AddProject<Projects.HelloAgent>("client")
+var client = builder.AddProject<Projects.HelloAgent>("HelloAgentsDotNET")
     .WithReference(backend)
-    .WithEnvironment("AGENT_HOST", $"{backend.GetEndpoint("https").Property(EndpointProperty.Url)}")
+    .WithEnvironment("AGENT_HOST", backend.GetEndpoint("https"))
+    .WithEnvironment("STAY_ALIVE_ON_GOODBYE", "true")
     .WaitFor(backend);
-
+#pragma warning disable ASPIREHOSTINGPYTHON001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+// xlang is over http for now - in prod use TLS between containers
+builder.AddPythonApp("HelloAgentsPython", "../../../../python/packages/autogen-core/samples/xlang/hello_python_agent", "hello_python_agent.py", "../../../../../.venv")
```

Comment:
How to setup `.venv`

Reply:
have to follow instructions at base python readme - will doc this in folow-up pr #4480